tests/Jwt/FirebaseParserTest.php

```diff
@@ -51,32 +50,15 @@ protected function setUp()
         ];
     }
 
+    /**
+     * @expectedException \Equip\Auth\Exception\InvalidException
```

Comment:
@shadowhand I like the new best practice of putting it in the code still, but at the top of the method.